mapie/metrics.py

```diff
@@ -737,3 +737,255 @@ def hsic(
     coef_hsic = np.sqrt(np.matrix.trace(hsic_mat, axis1=1, axis2=2))
 
     return coef_hsic
+
+
+def _picp(
+    y_true: ArrayLike,
+    y_pred_low: ArrayLike,
+    y_pred_up: ArrayLike
+) -> float:
+    """
+    Calculate the Prediction Interval Coverage Probability (PICP).
+
+    The Prediction Interval Coverage Probability (PICP) is a measure used to
+    estimate the fraction of true labels that lie within the prediction
+    intervals.
+
+    [5] Vilde Jensen, Filippo Maria Bianchi, Stian Norman Anfinsen (2022).
+    Ensemble Conformalized Quantile Regression for Probabilistic Time Series
+    Forecasting.
```

Comment:
@QMaphan, are you sure that this paper is the first one that proposes this metric?